packages/core-components/src/lib/unsorted/viz/ReferenceArea.svelte

```diff
@@ -194,6 +194,34 @@
 			return d;
 		});
 	}
+
+	$: chartOverrides = {
```

Comment:
I may be missing some context on how some of the charting logic works; but why does this need to be a reactive statement? From usage in this file it looks like it could be const

Reply:
It may not need to be, but that's how it's set up elsewhere - I pulled this directly from `Bar.svelte`